### ISSUE TITLE
CRITICAL ERROR: Wrong error input. Please re-install Nemesis

### ISSUE THREAD

**Issue by MooneShadow** [ShikyoKira/Project-New-Reign---Nemesis-Main#493]
Hello, 
I just moved my data from a D drive to a G drive after adding an NVME 2TB drive. 
I fixed all short cuts and ran the Nemesis only to come up with this error. Everything works except Nemesis. 
The two errors down there are from a magic mod the mod author stated to ignore these two errors. I know what they are. But the Critical Error i am unaware of. I have reinstalled and tried to do everything to fix this issue. Please Help. 

Checking engine version
Engine is up to date
Version Code: 8039-4585

Initializing behavior generation

Mod Checked 1: nemesis
Mod Checked 2: agpo
Mod Checked 3: flinch
Mod Checked 4: itmuse
Mod Checked 5: jpbo
Mod Checked 6: slide
Mod Checked 7: tkuc
Mod Checked 8: zhit

Mod installed: AnimatedWeddingScene
Mod installed: CoverSelf
Mod installed: FNIS_PCEA2
Mod installed: FNISSpells
Mod installed: Hoamaii_SkyrimKiss
Mod installed: Hoamaii_SkyrimWaltz
Mod installed: ImmersiveAnimatedLooting
Mod installed: JZBai_ThrowingWpnsLite
Mod installed: P1FlyingRing
Mod installed: RideSharing
Mod installed: SexLab
Mod installed: SexLabAP
Mod installed: SexLabCreature
Mod installed: SSM
Mod installed: TSC
Mod installed: XPMSE

CRITICAL ERROR: Wrong error input. Please re-install Nemesis

CRITICAL ERROR: Wrong error input. Please re-install Nemesis

ERROR(3020): Invalid format for "animationdatasinglefile.txt" in InfoData section has been detected. Please contact the template creator
Project: DefaultFemale.txt
Unique Code: <3>

WARNING(1027): 32bit hkx file detected. This file is not supported by current skyrim's architecture
File: G:\Steam\steamapps\common\Skyrim Special Edition\Data\meshes\actors\Character\animations\DynamicAnimationReplacer\_CustomConditions\70100\1hm_idle.hkx

WARNING(1027): 32bit hkx file detected. This file is not supported by current skyrim's architecture
File: G:\Steam\steamapps\common\Skyrim Special Edition\Data\meshes\actors\Character\animations\DynamicAnimationReplacer\_CustomConditions\998\1hm_idle.hkx

Failed to generate behavior

**Comment by AMGarkin** [comment]
32bit hkx file causes crashes with SSE. Either use mods designed for SSE (64bit) or at least convert old 32bit files to 64bit. (You can use for example [Cathedral Assets Optimizer](https://www.nexusmods.com/skyrimspecialedition/mods/23316))

**Comment by MooneShadow** [comment]
Yes but what about the critical error 3020. The other i can fix. I have disabled all my mods and still get the 3020.

**Comment by MooneShadow** [comment]
It seems stuck when trying to regenerate the files. I have my old files so slyrim is working but worried that this critical tool is not available if I run into any animation mods or trouble with animations.

**Comment by Naviorem** [comment]
can confirm that this works, thanks for the help

**Comment by WilsonBFerreira** [comment]
Hi, mine is giving this error, does anyone know how to solve it?


Checking engine version
Engine is up to date
Version Code: 2401-4678

Initializing behavior generation

Mod Checked 1: ostim
Mod Checked 2: oanims
Mod Checked 3: nemesis
Mod Checked 4: colis
Mod Checked 5: tdmh
Mod Checked 6: tdmlen
Mod Checked 7: tdmv

Mod installed: FlowerGirlsSE
Mod installed: FNIS_PCEA2
Mod installed: OSA
Mod installed: OSexVanillaDoggy
Mod installed: OSexVanillaKneeling
Mod installed: OSexVanillaMissionary
Mod installed: OSexVanillaStanding
Mod installed: OSexVanillaTumble
Mod installed: SMSkyrim

ERROR(3005): Invalid format for "animationdatasinglefile.txt" in AnimData section has been detected. Please contact the template creator
Project: ChickenProject.txt
Header: Forward_WalkSlow

Failed to generate behavior
